spring-integration-core/src/main/java/org/springframework/integration/endpoint/AbstractPollingEndpoint.java

```diff
@@ -66,23 +74,27 @@
 
 	private boolean syncExecutor = true;
 
+	private ClassLoader beanClassLoader = ClassUtils.getDefaultClassLoader();
+
+	private Trigger trigger = new PeriodicTrigger(10);
```

Comment:
Default 10ms? Really?

If we **must** have a default, it should be much larger than this.

Why wouldn't we just say it's invalid to configure a reactive poller without a trigger?

Reply:
Huh? This is an existing code. I just reformatted it a bit to remove some `volatile`.
We can reconsider the default in other issue, although I think Mark did this way to make as close to the event-driven behavior as possible

Reply:
Doh; merging.